src/Language/Mulang.hs

```diff
@@ -45,6 +45,7 @@ data Expression
         = TypeAliasDeclaration Identifier                    -- ^ Functional programming type alias. Only the type alias identifier is parsed 
         | RecordDeclaration Identifier                       -- ^ Imperative / Functional programming struct declaration. Only the record name is parsed
         | TypeSignature Identifier                           -- ^ Generic type signature for a computation. Only the target name of the computation is parsed
+        | Program Expression
```

Comment:
We should rename this element from Program to `EntryPoint`. 

We could also add some documentation like in the other constructors.